libs/blocks/global-navigation/utilities/utilities.js

```diff
@@ -307,11 +308,20 @@ export function trigger({ element, event, type } = {}) {
 export const yieldToMain = () => new Promise((resolve) => { setTimeout(resolve, 0); });
 
 export async function fetchAndProcessPlainHtml({ url, shouldDecorateLinks = true } = {}) {
+  const { mep } = getConfig();
+
   const path = getFederatedUrl(url);
   const res = await fetch(path.replace(/(\.html$|$)/, '.plain.html'));
   const text = await res.text();
   const { body } = new DOMParser().parseFromString(text, 'text/html');
 
+  if (mep.globalnav.length) {
```

Comment:
Will mep & mep.globalnav be always available or do we need to add a safety net? `mep?.globalnav?.length`

Reply:
Good catch. They should be available AFAICT (I've not had any errors while testing), but I'm gonna add the optional chaining in anyway.